src/components/Template/Template.mdx

```diff
@@ -123,6 +351,211 @@ If you are using `sidebar="right"`, place the `TemplateSidebar` after.
+    />
+  </Box>
+</Box>
+
+}
+
+## Understanding the Fluid Layout
+
+- `TemplateSidebar` and `TemplateMain` use fluid sizing where each fills a
+  fraction of the full width
+- Size fractions are dependent on viewport
+- Content in `TemplateMain`, especially grids, should adhere
+  to the <Link href="/?path=/docs/style-guide-responsive-grid--docs">
+  responsive grid</Link> recommendations and patterns
+
+**NOTE:** In the examples below, `TemplateSidebar` is highlighted in blue and
+`TemplateMain` is highlighted in purple.
+
+{
+
+<Box
+  display={{ base: "block", md: "grid" }}
+  gridTemplateColumns="1fr 1fr 1fr 1fr"
+  gap="l"
+>
+  <Box>
+    <Heading level="h3" size="heading7">
+      Small Mobile
+    </Heading>
+    <Image
+      alt="Fluid layout, 1 column"
+      border="1px solid var(--nypl-colors-ui-border-default)"
+      src="/template/templateFluidColumns1.png"
+      {...exampleWrapperStyles}
+      mb="s"
+    />
+    <Box>
+      <strong>Sidebar:</strong> 1/1 (100%))
+      <br />
+      <strong>Main:</strong> 1/1 (100%))
+    </Box>
+  </Box>
+  <Box>
+    <Heading level="h3" size="heading7">
+      Large Mobile
+    </Heading>
+    <Image
+      alt="Fluid layout, 2 columns"
+      border="1px solid var(--nypl-colors-ui-border-default)"
+      src="/template/templateFluidColumns2.png"
+      {...exampleWrapperStyles}
+      mb="s"
+    />
+    <Box>
+      <strong>Sidebar:</strong> 1/2 (50%)
+      <br />
+      <strong>Main:</strong> 1/2 (50%)
+    </Box>
+  </Box>
+  <Box>
+    <Heading level="h3" size="heading7">
+      Small Tablet
+    </Heading>
+    <Image
+      alt="Fluid layout, 3 columns"
+      border="1px solid var(--nypl-colors-ui-border-default)"
+      src="/template/templateFluidColumns3.png"
+      {...exampleWrapperStyles}
+      mb="s"
+    />
+    <Box>
+      <strong>Sidebar:</strong> 1/3 (33.3%)
+      <br />
+      <strong>Main:</strong> 2/3 (66.6%)
+    </Box>
+  </Box>
+  <Box>
+    <Heading level="h3" size="heading7">
+      Large Tablet and up
+    </Heading>
+    <Image
+      alt="Fluid layout, 4 columns"
+      border="1px solid var(--nypl-colors-ui-border-default)"
+      src="/template/templateFluidColumns4.png"
+      {...exampleWrapperStyles}
+      mb="s"
+    />
+    <Box>
+      <strong>Sidebar:</strong> 1/4 (25%)
+      <br />
+      <strong>Main:</strong> 3/4 (75%)
+    </Box>
+  </Box>
+</Box>
+}
+
+## Full Page Composition
+
+{
+
+<DocsWithImage
+  docs={
+    <>
+      <List type="ul">
+        <li>
+          The <code>Tempate</code> component should be used with the NYPL
+          universal <code>Header</code> and <code>Footer</code> to compose pages
+          aligned with NYPL design and application standards
+        </li>
+        <li>
+          If an app needs to render the <code>SkipNavigation</code> component,
+          it should be to be explicitly rendered immediately before the{" "}
+          <code>Template</code> component
+        </li>
```

Comment:
I think you should add that this is the general recommended shape of the DOM but it will differ based on each individual app's framework architecture, e.g. Nextjs's page and app router are slightly different.